nncf/openvino/quantization/quantize_model.py

```diff
@@ -308,6 +309,8 @@ def quantize_impl(
     """
     Implementation of the `quantize()` method for the OpenVINO backend.
     """
+    model = remove_friendly_name_duplicates(model)
```

Comment:
@kshpv, could you please ask if we should also check duplicates for the **if** bodies?

Reply:
Ask whom?

Reply:
Could you please say if should we check duplicates for the **if** bodies?